src/models/app.go

```diff
@@ -7,3 +7,23 @@ type AppInfo struct {
 type AppEntity struct {
 	Instances int `json:"instances"`
 }
+
+const (
+	ScalingTypeDynamic     = 0
```

Comment:
make ScalingType and ScalingStatus a type, e.g.,
type ScalingType int

you can use iota to automatically assign values or continue with giving it a value.